django/contrib/admin/options.py

```diff
@@ -642,7 +642,7 @@ def media(self):
         if self.actions is not None:
             js.append('actions%s.js' % extra)
         if self.prepopulated_fields:
-            js.extend(['urlify.js', 'prepopulate%s.js' % extra])
+            js.extend(['xregexp.min.js', 'urlify.js', 'prepopulate%s.js' % extra])
```

Comment:
Needs a rebase (these files are now alway included as of 28e8c54d7d2800a8ac98cf646a9e074055bc1b1b).